docs/api/auto-updater.md

```diff
@@ -50,15 +41,6 @@ The installer generated with Squirrel will create a shortcut icon with an
 same ID for your app with `app.setAppUserModelId` API, otherwise Windows will
 not be able to pin your app properly in task bar.
 
-Unlike Squirrel.Mac, Windows can host updates on S3 or any other static file host.
```

Comment:
I thought it wasn't necessary, but now - after thinking about it a bit more - I brought it back just to be sure that we're all on the same page: 3405596983ab7e78b229fba2cf16cc040549a367